library/src/main/java/com/liulishuo/filedownloader/FileDownloadMessageStation.java

```diff
@@ -102,6 +96,25 @@ private void handoverInUIThread(IFileDownloadMessenger messenger) {
         handler.sendMessage(handler.obtainMessage(HANDOVER_A_MESSENGER, messenger));
     }
 
+    /**
```

Comment:
what about `interceptForBlockCompleteMessage`? I think logic is not changed?

Reply:
Ok, this name is better.

Reply:
Updated.